Update README.md

Samsomyajit opened the pull request:
This update is requested in reference to the issue #33552 
[Under the heading "Obtain and Import the Library", the link incorrectly refers to the micro_speech.zip package instead of a person_detection example.]
So I updated the "Obtain and Import the Library" section on  the micro_speech.zip [now: person_detection_example.zip].

Samsomyajit committed:
Update README.md

This update is requested in reference to the issue #33552 
[Under the heading "Obtain and Import the Library", the link incorrectly refers to the micro_speech.zip package instead of a person_detection example.]
So I updated the "Obtain and Import the Library" section on  the micro_speech.zip [now: person_detection_example.zip].

Samsomyajit committed:
Update README.md

This update is requested in reference to the issue #33552
[Under the heading "Obtain and Import the Library", the link incorrectly refers to the micro_speech.zip package instead of a person_detection example.]
So I updated the "Obtain and Import the Library" section on the micro_speech.zip [now: person_detection.zip].

rthadur commented:
please combine #33582 with this PR

Samsomyajit commented:
in reference to issue #33552 .... Just like the person detection .zip name in the magic-wand documentation the same problem is present.

Samsomyajit commented:
@rthadur would you please check this out: when I click on the person_detection.zip link after update this pops out:

`
This XML file does not appear to have any style information associated with it. The document tree is shown below.
<Error nighteye="disabled">
<Code>NoSuchKey</Code>
<Message>The specified key does not exist.</Message>
<Details>
No such object: tensorflow-nightly/github/tensorflow/tensorflow/lite/experimental/micro/tools/make/gen/arduino_x86_64/prj/person_detection/person_detection.zip
</Details>
</Error>
`

Samsomyajit commented:
@rthadur Why close #33582 ? The magic_wand has the same problem as person_detection in README.md

Samsomyajit committed:
Update README.md

Actually got the link from @smellslikeml .... This is the updated link to the person_detection package zip..

Samsomyajit commented:
@rthadur Please check the update made to the README.md and tell me if its fine now

mihaimaruseac commented:
This should include the changes from #33582?

adisakshya reviewed (commented):
Yes, the link for ```person_detection``` is fixed. Also the```magic-wand``` documentation needs this same fix as #33582 suggests. You can include the changes from #33582 here in #33579.

Samsomyajit committed:
Update README.md

> Yes, the link for `person_detection` is fixed. Also the`magic-wand` documentation needs this same fix as #33582 suggests. You can include the changes from #33582 here in #33579.

In reverence to the above statement please update README.md for magic_wand

Samsomyajit commented:
> Yes, the link for `person_detection` is fixed. Also the`magic-wand` documentation needs this same fix as #33582 suggests. You can include the changes from #33582 here in #33579.

I have added the magic_wand.zip issue just like the person_detection as requested....

mihaimaruseac commented on tensorflow/lite/experimental/micro/examples/magic_wand/README.md:
````diff
@@ -346,6 +346,7 @@ SLOPE:
   *
  * * * * * * * *
 ```
+> Yes, the link for `person_detection` is fixed. Also the`magic-wand` documentation needs this same fix as #33582 suggests. You can include the changes from #33582 here in #33579.
````
Please remove this line

Samsomyajit committed:
Update README.md

The requested line is removed. plz have a look at it

Samsomyajit commented:
@mihaimaruseac PLease check the updated README.md the rquested line is removed.

Samsomyajit committed:
Merge branch 'master' into patch-1

mihaimaruseac commented on tensorflow/lite/experimental/micro/examples/magic_wand/README.md:
```diff
@@ -26,10 +26,17 @@ The sample has been tested with the following devices:
 
 ### Install the Arduino_TensorFlowLite library
 
+
```
Remove this additional empty line

mihaimaruseac commented on tensorflow/lite/experimental/micro/examples/magic_wand/README.md:
```diff
@@ -26,10 +26,17 @@ The sample has been tested with the following devices:
 Arduino library. To install it, open the Arduino library manager in
 `Tools -> Manage Libraries...` and search for `Arduino_TensorFlowLite`.
 
+
```
Remove this additional empty line

Samsomyajit replied:
Plz check! I made the necessary changes

mihaimaruseac commented on tensorflow/lite/experimental/micro/examples/person_detection/README.md:
```diff
@@ -42,10 +42,14 @@ Connect the Arducam pins as follows:
 
 ### Install the Arduino_TensorFlowLite library
 
+
```
Remove this additional empty line

mihaimaruseac commented on tensorflow/lite/experimental/micro/examples/person_detection/README.md:
```diff
@@ -42,10 +42,14 @@ Connect the Arducam pins as follows:
 Arduino library. To install it, open the Arduino library manager in
 `Tools -> Manage Libraries...` and search for `Arduino_TensorFlowLite`.
 
+
```
Remove this additional empty line

Samsomyajit committed:
Update README.md

Samsomyajit committed:
Update README.md

mihaimaruseac commented on tensorflow/lite/experimental/micro/examples/magic_wand/README.md:
```diff
@@ -26,6 +26,11 @@ The sample has been tested with the following devices:
+Download the current nightly build of the library: [magic_wand.zip](https://storage.googleapis.com/tensorflow-nightly/github/tensorflow/tensorflow/lite/experimental/micro/tools/make/gen/arduino_x86_64/prj/magic_wand/magic_wand.zip)
+
+Next, import this zip file into the Arduino Desktop IDE by going to `Sketch ->
+Include Library -> Add .ZIP Library...`.
```
`Sketch -> ..` should all be on a single line.

Samsomyajit committed:
Update README.md

Shusuke-O commented:
Hello. I'm trying to check a magic wand example. But I cannot download magic_wand.zip on the README.md.
